scripts/backup

```diff
@@ -51,6 +52,10 @@ ynh_backup --src_path="$final_path"
 ynh_script_progression --message="Backing up nginx web server configuration..." --weight=1
 
 ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.conf"
+if yunohost --output-as plain domain list | grep -q "^$server_name$"
+then
+    ynh_backup --src_path="/etc/nginx/conf.d/${server_name}.d/server_name.conf"
```

Comment:
Now I'm thinking that we should add the app name in the file name, to avoid any conflict. So I suggest this:
```suggestion
    ynh_backup --src_path="/etc/nginx/conf.d/${server_name}.d/${app}_server_name.conf"
```